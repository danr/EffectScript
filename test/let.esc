x = 1
y = 2
z = 3
type T2<a,b> { T2(a,b) }
main = { T2(x, y + z) }
