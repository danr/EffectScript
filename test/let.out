T2(1, 5)
